.github/workflows/phpunit-tests.yml

```
@@ -36,7 +36,7 @@ jobs:
   #
   test-with-mysql:
     name: PHP ${{ matrix.php }}
-    uses: WordPress/wordpress-develop/.github/workflows/phpunit-tests-run.yml@trunk
+    uses: youknowriad/wordpress-develop/.github/workflows/phpunit-tests-run.yml@update/packages-65
```

This can also be used for testing (personally, I just find it more reliable).

```suggestion
    uses: ./.github/workflows/phpunit-tests-run.yml
```

---

yes, this is being discussed here https://wordpress.slack.com/archives/C02RQBWTW/p1706523051675219 and we may do these changes separately (in their own ticket/PR)